### ISSUE TITLE
Pre-trained model file?

### ISSUE THREAD

**Issue by musab1234** [RoyalStorm/nails-tracker#17]
Can i have pretrained mask rcnn file or dataset, for testing this repo?

**Comment by RoyalStorm** [comment]
We do not provide the model itself, since this is the private part. You can train Mask R-CNN on your own using your labeled data.
